solutions/instances/variables.tf

```diff
@@ -90,6 +96,12 @@ variable "log_archive_api_key" {
 # Cloud Monitoring Variables
 ##############################################################################
 
+variable "cloud_monitoring_provision" {
+  description = "Provision a IBM cloud monitoring instance?"
```

Comment:
Provide a better description instead of having `?` at the end

Reply:
changed